web/sites/default/config/embed.button.node.yml

```diff
@@ -15,3 +15,4 @@ type_settings:
   bundles: {  }
   display_plugins: {  }
 icon_uuid: null
+icon_path: null
```

Comment:
Does this mean we need to fix the references to the embed buttons after deployment to live and then copy that down to local to update the config post. We should add that to the deploy notes.